README.md

```diff
@@ -1,8 +1,37 @@
-# epuck_ros2
+# E-Puck Driver for ROS2
 [![Build Status](https://travis-ci.com/cyberbotics/epuck_ros2.svg?branch=master)](https://travis-ci.com/cyberbotics/epuck_ros2)
 [![license - apache 2.0](https://img.shields.io/:license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Version](https://img.shields.io/github/v/tag/cyberbotics/epuck_ros2?label=version)](http://wiki.ros.org/epuck_ros2)
 
+<img src="./assets/pi-puck.jpg" width="400px" />
+
+This ROS2 package adds support for [e-puck](https://www.gctronic.com/doc/index.php/e-puck2) physical robot with [Pi-puck extension](https://www.gctronic.com/doc/index.php?title=Pi-puck).
+Please use the following links for the instructions:
+- [Installation](https://github.com/cyberbotics/epuck_ros2/tree/master/installation)
+- [Getting Started](https://github.com/cyberbotics/epuck_ros2#getting-started)
```

Comment:
```suggestion
- [Getting Started](#getting-started)
```

Not sure, but worth to try.

Reply:
I put the full link in case it is used to generate external documentation, but actually it the exact reason NOT to use full links. Thanks!